developer-docs-site/docs/guides/move-guides/move-on-aptos.md

```diff
@@ -114,19 +114,19 @@ public fun deposit<T>(account: address, coin: Coin<T>)
 
 ### Type-safety
 
-In Move, given a specific struct, say `A`, different instances can be made distinct in two fashions:
+In Move, given a specific struct, say `A`, different instances can be made distinct in two ways:
 * Internal identifiers, such as [GUID](https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/aptos-stdlib/sources/guid.move)s
 * Generics such as `A<T>`, where `T` is another struct
 
 Internal identifiers can be convenient due to their simplicity and easier programmability. Generics, however, provide much higher guarantees including explicit compile or validation time checks though with some costs.
 
-Generics allow for completely distinct types and resources and interfaces that expects those types. For example, an order book can state that they expect two currencies for all orders but one of them must be fixed, e.g., `buy<T>(coin: Coin<Aptos>): Coin<T>`. This explicitly states that a user can buy any coin `<T>` but must pay for it with `Coin<Aptos>`.
+Generics allow for completely distinct types, and resources and interfaces that expects those types. For example, an order book can permit orders between two currencies, but with one currency fixed, e.g., `buy<T>(coin: Coin<Aptos>): Coin<T>`. This explicitly states that a user can buy any coin `<T>` but must pay for it with `Coin<Aptos>`.
```

Comment:
Ah, I am not sure if this fix is equivalent. Does the phrase "an order book can permit orders between two currencies, but with one currency fixed" convey the same meaning as "an order book can state that they expect two currencies for all orders but one of them must be fixed"? Methinks no. What do you think, @ramiksadana?

Reply:
I'm interpreting the code that follows, and it seems weird to say that an API 'states' anything. But feel free to revert it to the original. It can even be `For example, an order book can state that they expect two currencies for all orders, but with one currency fixed`.